elasticsearch/run.sh

```diff
@@ -60,4 +57,18 @@ else
 	exit 1
 fi
 
-exec /usr/share/elasticsearch/bin/elasticsearch
+TIMES=60
```

Comment:
@sosiouxme i removed the `exit 255` since it probably isn't doing what i think/want it to do.
I also bumped us up to 300 from 60; I considered a do-while but I didn't want this out these spinning forever if someone ES gets stuck...